core/templates/frontend_init.php

```diff
@@ -84,9 +97,12 @@
     }
 
     $api_verification = $configuration->get('Core', 'api_verification');
-    if ($api_verification == 1 && in_array($default_group, $user->getAllGroupIds()) && ($user->data()->reset_code)) {
-        // User needs to validate account
-        $smarty->assign('MUST_VALIDATE_ACCOUNT', str_replace('{x}', Output::getClean($user->data()->reset_code), $language->get('user', 'validate_account_command')));
+    if ($api_verification == 1) {
+        $integrationUser = $user->getIntegration('Minecraft');
```

Comment:
does this mean that its only possible to require them to verifiy their minecraft account? they can't make them verify a discord account or different integration?

Reply:
They can require any integration, so that banner is currently only used when its not required so ask to verify account, guess we can remove the banner